tailcfg/tailcfg.go

```diff
@@ -1503,6 +1504,9 @@ const (
 	// CapabilityFileSharingSend grants the ability to receive files from a
 	// node that's owned by a different user.
 	CapabilityFileSharingSend = "https://tailscale.com/cap/file-send"
+	// CapabilityFileSharingSendTo grants the ability to send files to a
+	// node that's owned by a different user.
```

Comment:
this isn't worded clearly.

It almost sounds backwards.

Reply:
reworded, PTAL

Reply:
also moved it to the other capability section to be clear that its not in the filter but in`Node.Capabilities`